### ISSUE TITLE
Redesign Landing Page

### ISSUE THREAD

**Issue by jmash** [cforlando/Daily-Spoonie#10]
Contrast ratio check shows that the colors being used for the background and the color being used for the font are not WCAG compliant (https://contrast-ratio.com/#white-on-%2331c4f5 and https://contrast-ratio.com/#white-on-%23169BD5). Component should be better designed for color contrast and accessibility compliance.
